src/middleware.ts

```diff
@@ -169,23 +166,9 @@ export async function middleware(request: NextRequest) {
             userAgent: userAgent(),
         }),
         async () => {
-            // Start fetching everything as soon as possible, but do not block the middleware on it
-            // the cache will handle concurrent calls
-            await waitUntil(
-                getSpaceContentData({
-                    spaceId: resolved.space,
-                    changeRequestId: resolved.changeRequest,
-                    revisionId: resolved.revision,
-                }),
-            );
```

Comment:
I don't understand why you removed it and called it useless. I measured it at the time, and it was efficient.

It doesn't block anything because: we don't await it, we only start the execution asap, and since the cache is maintaining only one operation per cache key, it's being reused when needed during rendering